### DISCUSSION TITLE
User page - reservations for the conditioned logged-in user

### DISCUSSION THREAD

**Discussion by tiicaa** [liedekef/events-made-easy#147] [Q&A]
Hello, I was wondering if there is a parameter to use in [eme_mybookings] to show only if they are event records 6,7,8,or 9. Could I use something like this that works? [eme_if event_ids="6,7,8,9"][eme_mybookings template_id=4][/eme_if] I tried something like that too [eme_mybookings ids=6,7,8,9 template_id=4] but it didn't work, spi works if I only put

How could I do that?

**Comment by liedekef** [comment]
using the shortcode [eme_events] you can limit the list to certain event ids. Then use a template for the "content" of the event, and that template can either contain #_BOOKINGS or  the shortcode [eme_bookings] with as id-option #_EVENTID, see https://www.e-dynamics.be/wordpress/category/documentation/6-shortcodes/eme_bookings/

**Comment by tiicaa** [comment]
@liedekef I appreciate your return.
But in this case, what I need is to see the logged in user, the reservations he made. I have a visualization model that is specific for 4 types of events, but the user can only register in 1 event of these 3, that is, the choice is optional, or the user chooses 1 or chooses 2 or chooses event 3, after it filling out the RSVP form he is directed to a page called "My Account" there should appear all the registrations in events, however there is a space with a model made for [eme_mybookings] and in this place only one of the 4 options of events that he registered .
in the other event records I'm calling the shortcode like this: [eme_**mybookings**  id=4 template_id=2 ] in this example is the registration that the user made only in event id 4 calling template 2 , this works for me.
But what I need is something like this: [eme_mybookings **ids=5,6,7,8** template_id=3 ] - is the registration in one of the events of ids 5,6,7 or 8 calling template 3, however "**ids = 5,6,7,8**" is not working and I still can't find anything in the docs that I can put there.

**Comment by liedekef** [comment]
Then use my advice but use [eme_mybookings id=#_EVENTID]

**Comment by tiicaa** [comment]
Sorry for the questions, but I'm having trouble understanding what your guidelines are because of the language.
So what you are proposing to me would be to create a template and in that template add the shortcode [eme_mybookings id=#_EVENTID] then go there in the events with ids 4,5 and 6 and assign in the section Events -> Single Event -> model that was made in each of the mentioned events. Then on the "My Account" page add the shortcode [eme_mybookings template=2]
would it be this?
But how would that solve the problem, because when the user enters the site and sees the events of ids 4,5 and 6 he must randomly choose one of them, register and when he goes to his account page he will only see what he did to choose?

**Comment by liedekef** [comment]
no, you create that template and add it as an option to [eme_events] using the template_id option. To limit to the events of choice, add a category to those events and then add the category option to the eme_events shortcode as well.

**Comment by tiicaa** [comment]
@liedekef Thanks for the feedback, but that's not quite what I need. I just want to limit the view of the registration by the user on the page where the mybookings shortcode goes, I just want the record of one of the events that he chose to appear. But I don't know which record that would be but I know that it would only be once in one of these events, so the template that I call in mybookings is specific for each event so the other events have different mybookings templates, but I need to limit the visualization of these that I want because otherwise all the records will appear in the places I don't want. For example, each layout I delimited by the id of the event which record should appear, but I came to a problem where it is not just an id but 4 how could I call it I thought I could use id=1,2,3,4 like this separately by comma but it doesn't work, I tried to use category_id and put the category of events because it's the same for these 4 but it didn't work either. Can't I use this in the mybookings shortcode? Is there another call that I missed?

**Reply by liedekef** [reply]
then use [eme_mybookings] inside the single event template.
[eme_bookings] is to show a list of bookings for the logged in user, optionally limited to 1 event. Using that inside [eme_events] you can then limit to whatever you want and show it in any way you want. I don't understand what is not possible then ...
